### PR TITLE
Mention unregistering device when logout

### PR DESCRIPTION
**Pull request by b123400**
connect https://github.com/SkygearIO/skygear-SDK-iOS/issues/76

### CODE REVIEW

**Comment by oursky-travis** [comment]
@b123400, thanks for your PR! By analyzing the history of the files in this pull request, we identified @bensonby to be a potential reviewer.

**Review by cheungpat — APPROVED** [review]

**Comment by cheungpat** [comment]
I am okay with the code sample, and I will merge this if @bensonby have no problem with the technical writing and when SkygearIO/skygear-SDK-iOS#76 is merged.

File: push-notifications/basics/ios.md [hunk 1]
```diff
@@ -202,6 +202,34 @@ operation?.sendCompletionHandler = { (deviceIDs, error) in
 }
 ```
 
+## Unregistering device
+
+When a device is registered, it is associated with the user. Therefore unregister is needed when the user is no longer associated with the device, for example when the user logouts.
```
Comment (bensonby) [inline push-notifications/basics/ios.md]: @b123400 I have some comments for this sentence:

- Better break the sentence for the 80-character line limit, and
- "it is associated with the authenticated user"
- "Therefore it is necessary to unregister the device when" (See notes below)
- "when the user logs out."

I believe "deregister" is more appropriate from a pure English standpoint, but given that "unregister" is more popular we will stick with "unregister". However "unregistering" or "unregistration" does not sound natural to me

**Comment by cheungpat** [comment]
...really seems deregister is better...

**Comment by bensonby** [comment]
yes, yet after reading the discussion in http://english.stackexchange.com/questions/25931/unregister-vs-deregister, I understand it's true that unregister is more common at least in api doc, e.g. even "unregistration" is used in https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/unregister

**Comment by chpapa** [comment]
Wo thanks everyone for the research. So maybe we should go ahead with using unregister in API, and de-register with the content?

**Comment by cheungpat** [comment]
I am going to make the changes since they are trivial.

**Comment by cheungpat** [comment]
@b123400 I suppose you should make a pull request from a branch on your own repo instead of the official repo.

**Comment by b123400** [comment]
Oh yes, forgot to fork. I also think "deregister" is better, though I favour consistency (with method name) over accuracy.

**Pull request merged**